workspace/apps/pidp/src/app/features/portal/state/access/ms-teams-privacy-officer-portal-section.class.ts

```diff
@@ -32,6 +33,7 @@ export class MsTeamsPrivacyOfficerPortalSection implements IPortalSection {
     this.heading = 'MS Teams for Clinical Use';
     this.description = `Privacy Officer enrol here for clinic access for MS Teams with Fraser Health.`;
     this.keyWords = profileStatus.status.msTeamsPrivacyOfficer.keyWords || [];
+    this.completedMessage = this.profileStatus.status.msTeamsPrivacyOfficer.completedMessage;
```

Comment:
Since this is not a dynamic value, I think this could be hardcoded on the frontend like `heading`, `description`. 

For example, a dynamic value could be if this message was on display at all times, and its value changed. 

Something like `this.message = this.profileStatus.status.msTeamsPrivacyOfficer.message;`